sockeye/attention.py

```diff
@@ -315,6 +323,7 @@ def attend(att_input: AttentionInput, att_state: AttentionState) -> AttentionSta
                                               weight=self.t2h_weight,
                                               num_hidden=self.num_hidden,
                                               no_bias=True, name="%squery_hidden_fc" % self.prefix)
+            query *= self.num_hidden ** -0.5
```

Comment:
There's also a statement below to scale the query.  Is this applying scaling twice?